data/data/gcp/network/internal-lb.tf

```diff
@@ -0,0 +1,49 @@
+resource "google_compute_address" "cluster_private_ip" {
+  name = "${var.cluster_id}-cluster-private-ip"
+
+  address      = cidrhost(var.master_subnet_cidr, -3) #Last two addressed used by gcp.
+  address_type = "INTERNAL"
+  subnetwork   = google_compute_subnetwork.master_subnet.self_link
+}
+
+resource "google_compute_health_check" "api_internal" {
+  name = "${var.cluster_id}-api-internal"
+
+  https_health_check {
+    port         = 6443
+    request_path = "/readyz"
+  }
+}
+
+resource "google_compute_region_backend_service" "api_internal" {
+  name = "${var.cluster_id}-api-internal"
+
+  load_balancing_scheme = "INTERNAL"
+  protocol              = "TCP"
+  timeout_sec           = 120
+
+  backend {
+    group = var.bootstrap_instance_group
+  }
+
+  dynamic "backend" {
+    for_each = var.master_instance_groups
```

Comment:
why not `concat(var.master_instance_groups, [var.bootstrap_instance_group])`

Reply:
Good idea! Done.